tests/sles4sap/qesapdeployment/configure.pm

```diff
@@ -30,6 +30,12 @@ sub run {
     }
 
     my %variables;
+    $variables{TERRAFORM_RUNNER} = get_var('QESAPDEPLOY_TERRAFORM_RUNNER', 'terraform');
```

Comment:
Why runner and not binary?

Reply:
To partially remand to the name of the variable used in PC https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/f48ae20a2e51708ac40c5f23df4f2cdf7ca07c38/variables.md?plain=1#L406